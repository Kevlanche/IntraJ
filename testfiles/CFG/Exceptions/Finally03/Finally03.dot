digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[6]"; "Opt[9]"; "List[10]"; "List[11]"}
{ rank = same; "List[7]"; "MethodDecl[12]"}
{ rank = same; "Modifier[8]"; "Entry[13]"; "Modifiers[14]"; "PrimitiveTypeAccess[17]"; "List[18]"; "List[19]"; "Opt[20]"; "Exit[21]"}
{ rank = same; "List[15]"; "Block[21]"}
{ rank = same; "Modifier[16]"; "List[22]"}
{ rank = same; "TryStmt[23]"}
{ rank = same; "Block[24]"; "List[93]"; "Opt[94]"; "UncheckedExceptions[108]"}
{ rank = same; "List[25]"; "Block[95]"}
{ rank = same; "TryStmt[26]"; "List[96]"}
{ rank = same; "Block[27]"; "List[77]"; "Opt[78]"; "UncheckedExceptions[92]"; "ExprStmt[97]"}
{ rank = same; "List[28]"; "Block[79]"; "Dot[98]"}
{ rank = same; "ExprStmt[29]"; "Block[40]"; "List[80]"; "Dot[99]"; "MethodAccess[102]"}
{ rank = same; "Dot[30]"; "List[41]"; "ExprStmt[81]"; "TypeAccess[100]"; "VarAccess[101]"; "List[104]"; "List[106]"; "List[107]"; "UncheckedExceptions[103]"}
{ rank = same; "Dot[31]"; "MethodAccess[34]"; "ReturnStmt[42]"; "Dot[82]"; "StringLiteral[105]"}
{ rank = same; "TypeAccess[32]"; "VarAccess[33]"; "List[36]"; "List[38]"; "List[39]"; "UncheckedExceptions[35]"; "Opt[43]"; "List[76]"; "UncheckedExceptions[43]"; "Dot[83]"; "MethodAccess[86]"}
{ rank = same; "StringLiteral[37]"; "IntegerLiteral[44]"; "NTAFinallyBlock[46]"; "TypeAccess[84]"; "VarAccess[85]"; "List[88]"; "List[90]"; "List[91]"; "UncheckedExceptions[87]"}
{ rank = same; "List[47]"; "StringLiteral[89]"}
{ rank = same; "Block[48]"; "NTAFinallyBlock[61]"}
{ rank = same; "List[49]"; "List[62]"}
{ rank = same; "ExprStmt[50]"; "Block[63]"}
{ rank = same; "Dot[51]"; "List[64]"}
{ rank = same; "Dot[52]"; "MethodAccess[55]"; "ExprStmt[65]"}
{ rank = same; "TypeAccess[53]"; "VarAccess[54]"; "List[57]"; "List[59]"; "List[60]"; "UncheckedExceptions[56]"; "Dot[66]"}
{ rank = same; "StringLiteral[58]"; "Dot[67]"; "MethodAccess[70]"}
{ rank = same; "TypeAccess[68]"; "VarAccess[69]"; "List[72]"; "List[74]"; "List[75]"; "UncheckedExceptions[71]"}
{ rank = same; "StringLiteral[73]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Finally03 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[8]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[9]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[12]" [label="MethodDecl\n < fun >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[13]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[14]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[15]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[16]" [label="Modifier\n < static >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[17]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[18]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[19]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[20]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[21]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[22]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TryStmt[23]" [label="TryStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[24]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[25]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TryStmt[26]" [label="TryStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[27]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[28]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[29]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[30]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[31]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[32]" [label="TypeAccess\n < System >", fillcolor=white   style=filled  ]
"VarAccess[33]" [label="VarAccess\n < out >", fillcolor=white   style=filled  ]
"MethodAccess[34]" [label="MethodAccess\n < println >", fillcolor=white   style=filled  ]
"List[36]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"StringLiteral[37]" [label="StringLiteral\n < Try >", fillcolor=white   style=filled  ]
"List[38]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[39]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[35]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[40]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[41]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ReturnStmt[42]" [label="ReturnStmt\n ", fillcolor=white   style=filled  ]
"Opt[43]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[44]" [label="IntegerLiteral\n < 4 >", fillcolor=white   style=filled  ]
"NTAFinallyBlock[46]" [label="NTAFinallyBlock\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[47]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[48]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[49]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[50]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[51]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[52]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[53]" [label="TypeAccess\n < System >", fillcolor=white   style=filled  ]
"VarAccess[54]" [label="VarAccess\n < out >", fillcolor=white   style=filled  ]
"MethodAccess[55]" [label="MethodAccess\n < println >", fillcolor=white   style=filled  ]
"List[57]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"StringLiteral[58]" [label="StringLiteral\n < Finally1 >", fillcolor=white   style=filled  ]
"List[59]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[60]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[56]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"NTAFinallyBlock[61]" [label="NTAFinallyBlock\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[62]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[63]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[64]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[65]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[66]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[67]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[68]" [label="TypeAccess\n < System >", fillcolor=white   style=filled  ]
"VarAccess[69]" [label="VarAccess\n < out >", fillcolor=white   style=filled  ]
"MethodAccess[70]" [label="MethodAccess\n < println >", fillcolor=white   style=filled  ]
"List[72]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"StringLiteral[73]" [label="StringLiteral\n < Finally2 >", fillcolor=white   style=filled  ]
"List[74]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[75]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[71]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[76]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[43]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"List[77]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[78]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[79]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[80]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[81]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[82]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[83]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[84]" [label="TypeAccess\n < System >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarAccess[85]" [label="VarAccess\n < out >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodAccess[86]" [label="MethodAccess\n < println >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[88]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"StringLiteral[89]" [label="StringLiteral\n < Finally1 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[90]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[91]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[87]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[92]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[93]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[94]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[95]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[96]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[97]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[98]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[99]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[100]" [label="TypeAccess\n < System >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarAccess[101]" [label="VarAccess\n < out >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodAccess[102]" [label="MethodAccess\n < println >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[104]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"StringLiteral[105]" [label="StringLiteral\n < Finally2 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[106]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[107]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[103]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[108]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[21]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[7]" -> "Modifier[8]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[9]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Entry[13]" [style=dashed, color=gray]
"List[15]" -> "Modifier[16]" [style=dashed, color=gray]
"Modifiers[14]" -> "List[15]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Modifiers[14]" [style=dashed, color=gray]
"MethodDecl[12]" -> "PrimitiveTypeAccess[17]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[18]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[19]" [style=dashed, color=gray]
"Dot[31]" -> "TypeAccess[32]" [style=dashed, color=gray]
"Dot[31]" -> "VarAccess[33]" [style=dashed, color=gray]
"Dot[30]" -> "Dot[31]" [style=dashed, color=gray]
"List[36]" -> "StringLiteral[37]" [style=dashed, color=gray]
"MethodAccess[34]" -> "List[36]" [style=dashed, color=gray]
"MethodAccess[34]" -> "List[38]" [style=dashed, color=gray]
"MethodAccess[34]" -> "List[39]" [style=dashed, color=gray]
"MethodAccess[34]" -> "UncheckedExceptions[35]" [style=dashed, color=gray]
"Dot[30]" -> "MethodAccess[34]" [style=dashed, color=gray]
"ExprStmt[29]" -> "Dot[30]" [style=dashed, color=gray]
"List[28]" -> "ExprStmt[29]" [style=dashed, color=gray]
"Opt[43]" -> "IntegerLiteral[44]" [style=dashed, color=gray]
"ReturnStmt[42]" -> "Opt[43]" [style=dashed, color=gray]
"Dot[52]" -> "TypeAccess[53]" [style=dashed, color=gray]
"Dot[52]" -> "VarAccess[54]" [style=dashed, color=gray]
"Dot[51]" -> "Dot[52]" [style=dashed, color=gray]
"List[57]" -> "StringLiteral[58]" [style=dashed, color=gray]
"MethodAccess[55]" -> "List[57]" [style=dashed, color=gray]
"MethodAccess[55]" -> "List[59]" [style=dashed, color=gray]
"MethodAccess[55]" -> "List[60]" [style=dashed, color=gray]
"MethodAccess[55]" -> "UncheckedExceptions[56]" [style=dashed, color=gray]
"Dot[51]" -> "MethodAccess[55]" [style=dashed, color=gray]
"ExprStmt[50]" -> "Dot[51]" [style=dashed, color=gray]
"List[49]" -> "ExprStmt[50]" [style=dashed, color=gray]
"Block[48]" -> "List[49]" [style=dashed, color=gray]
"List[47]" -> "Block[48]" [style=dashed, color=gray]
"Dot[67]" -> "TypeAccess[68]" [style=dashed, color=gray]
"Dot[67]" -> "VarAccess[69]" [style=dashed, color=gray]
"Dot[66]" -> "Dot[67]" [style=dashed, color=gray]
"List[72]" -> "StringLiteral[73]" [style=dashed, color=gray]
"MethodAccess[70]" -> "List[72]" [style=dashed, color=gray]
"MethodAccess[70]" -> "List[74]" [style=dashed, color=gray]
"MethodAccess[70]" -> "List[75]" [style=dashed, color=gray]
"MethodAccess[70]" -> "UncheckedExceptions[71]" [style=dashed, color=gray]
"Dot[66]" -> "MethodAccess[70]" [style=dashed, color=gray]
"ExprStmt[65]" -> "Dot[66]" [style=dashed, color=gray]
"List[64]" -> "ExprStmt[65]" [style=dashed, color=gray]
"Block[63]" -> "List[64]" [style=dashed, color=gray]
"List[62]" -> "Block[63]" [style=dashed, color=gray]
"NTAFinallyBlock[61]" -> "List[62]" [style=dashed, color=gray]
"List[47]" -> "NTAFinallyBlock[61]" [style=dashed, color=gray]
"NTAFinallyBlock[46]" -> "List[47]" [style=dashed, color=gray]
"ReturnStmt[42]" -> "NTAFinallyBlock[46]" [style=dashed, color=gray]
"ReturnStmt[42]" -> "List[76]" [style=dashed, color=gray]
"ReturnStmt[42]" -> "UncheckedExceptions[43]" [style=dashed, color=gray]
"List[41]" -> "ReturnStmt[42]" [style=dashed, color=gray]
"Block[40]" -> "List[41]" [style=dashed, color=gray]
"List[28]" -> "Block[40]" [style=dashed, color=gray]
"Block[27]" -> "List[28]" [style=dashed, color=gray]
"TryStmt[26]" -> "Block[27]" [style=dashed, color=gray]
"TryStmt[26]" -> "List[77]" [style=dashed, color=gray]
"Dot[83]" -> "TypeAccess[84]" [style=dashed, color=gray]
"Dot[83]" -> "VarAccess[85]" [style=dashed, color=gray]
"Dot[82]" -> "Dot[83]" [style=dashed, color=gray]
"List[88]" -> "StringLiteral[89]" [style=dashed, color=gray]
"MethodAccess[86]" -> "List[88]" [style=dashed, color=gray]
"MethodAccess[86]" -> "List[90]" [style=dashed, color=gray]
"MethodAccess[86]" -> "List[91]" [style=dashed, color=gray]
"MethodAccess[86]" -> "UncheckedExceptions[87]" [style=dashed, color=gray]
"Dot[82]" -> "MethodAccess[86]" [style=dashed, color=gray]
"ExprStmt[81]" -> "Dot[82]" [style=dashed, color=gray]
"List[80]" -> "ExprStmt[81]" [style=dashed, color=gray]
"Block[79]" -> "List[80]" [style=dashed, color=gray]
"Opt[78]" -> "Block[79]" [style=dashed, color=gray]
"TryStmt[26]" -> "Opt[78]" [style=dashed, color=gray]
"TryStmt[26]" -> "UncheckedExceptions[92]" [style=dashed, color=gray]
"List[25]" -> "TryStmt[26]" [style=dashed, color=gray]
"Block[24]" -> "List[25]" [style=dashed, color=gray]
"TryStmt[23]" -> "Block[24]" [style=dashed, color=gray]
"TryStmt[23]" -> "List[93]" [style=dashed, color=gray]
"Dot[99]" -> "TypeAccess[100]" [style=dashed, color=gray]
"Dot[99]" -> "VarAccess[101]" [style=dashed, color=gray]
"Dot[98]" -> "Dot[99]" [style=dashed, color=gray]
"List[104]" -> "StringLiteral[105]" [style=dashed, color=gray]
"MethodAccess[102]" -> "List[104]" [style=dashed, color=gray]
"MethodAccess[102]" -> "List[106]" [style=dashed, color=gray]
"MethodAccess[102]" -> "List[107]" [style=dashed, color=gray]
"MethodAccess[102]" -> "UncheckedExceptions[103]" [style=dashed, color=gray]
"Dot[98]" -> "MethodAccess[102]" [style=dashed, color=gray]
"ExprStmt[97]" -> "Dot[98]" [style=dashed, color=gray]
"List[96]" -> "ExprStmt[97]" [style=dashed, color=gray]
"Block[95]" -> "List[96]" [style=dashed, color=gray]
"Opt[94]" -> "Block[95]" [style=dashed, color=gray]
"TryStmt[23]" -> "Opt[94]" [style=dashed, color=gray]
"TryStmt[23]" -> "UncheckedExceptions[108]" [style=dashed, color=gray]
"List[22]" -> "TryStmt[23]" [style=dashed, color=gray]
"Block[21]" -> "List[22]" [style=dashed, color=gray]
"Opt[20]" -> "Block[21]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Opt[20]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Exit[21]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[12]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[11]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[21]" -> "ReturnStmt[42]" [color=red, constraint=false]
"Exit[21]" -> "MethodAccess[70]" [color=red, constraint=false]
"ReturnStmt[42]" -> "Exit[21]" [color=blue, constraint=false, xlabel=" "] 
"ReturnStmt[42]" -> "MethodAccess[70]" [color=red, constraint=false]
"MethodAccess[70]" -> "ReturnStmt[42]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[70]" -> "Exit[21]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[70]" -> "StringLiteral[73]" [color=red, constraint=false]
"StringLiteral[73]" -> "MethodAccess[70]" [color=blue, constraint=false, xlabel=" "] 
"StringLiteral[73]" -> "VarAccess[69]" [color=red, constraint=false]
"VarAccess[69]" -> "StringLiteral[73]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[69]" -> "TypeAccess[68]" [color=red, constraint=false]
"TypeAccess[68]" -> "VarAccess[69]" [color=blue, constraint=false, xlabel=" "] 
"TypeAccess[68]" -> "MethodAccess[55]" [color=red, constraint=false]
"MethodAccess[55]" -> "TypeAccess[68]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[55]" -> "StringLiteral[58]" [color=red, constraint=false]
"StringLiteral[58]" -> "MethodAccess[55]" [color=blue, constraint=false, xlabel=" "] 
"StringLiteral[58]" -> "VarAccess[54]" [color=red, constraint=false]
"VarAccess[54]" -> "StringLiteral[58]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[54]" -> "TypeAccess[53]" [color=red, constraint=false]
"TypeAccess[53]" -> "VarAccess[54]" [color=blue, constraint=false, xlabel=" "] 
"TypeAccess[53]" -> "UncheckedExceptions[43]" [color=red, constraint=false]
"UncheckedExceptions[43]" -> "TypeAccess[53]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[43]" -> "IntegerLiteral[44]" [color=red, constraint=false]
"IntegerLiteral[44]" -> "UncheckedExceptions[43]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[44]" -> "MethodAccess[34]" [color=red, constraint=false]
"MethodAccess[34]" -> "IntegerLiteral[44]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[34]" -> "StringLiteral[37]" [color=red, constraint=false]
"StringLiteral[37]" -> "MethodAccess[34]" [color=blue, constraint=false, xlabel=" "] 
"StringLiteral[37]" -> "VarAccess[33]" [color=red, constraint=false]
"VarAccess[33]" -> "StringLiteral[37]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[33]" -> "TypeAccess[32]" [color=red, constraint=false]
"TypeAccess[32]" -> "VarAccess[33]" [color=blue, constraint=false, xlabel=" "] 
"TypeAccess[32]" -> "Entry[13]" [color=red, constraint=false]
"Entry[13]" -> "TypeAccess[32]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"Modifiers[6]" -> "Opt[9]" -> "List[10]" -> "List[11]" [style=invis]
"Entry[13]" -> "Modifiers[14]" -> "PrimitiveTypeAccess[17]" -> "List[18]" -> "List[19]" -> "Opt[20]" -> "Exit[21]" [style=invis]
"Block[24]" -> "List[93]" -> "Opt[94]" [style=invis]
"Block[27]" -> "List[77]" -> "Opt[78]" [style=invis]
"ExprStmt[29]" -> "Block[40]" [style=invis]
"Dot[31]" -> "MethodAccess[34]" [style=invis]
"TypeAccess[32]" -> "VarAccess[33]" [style=invis]
"Block[48]" -> "NTAFinallyBlock[61]" [style=invis]
"Dot[52]" -> "MethodAccess[55]" [style=invis]
"TypeAccess[53]" -> "VarAccess[54]" [style=invis]
"Dot[67]" -> "MethodAccess[70]" [style=invis]
"TypeAccess[68]" -> "VarAccess[69]" [style=invis]
"Dot[83]" -> "MethodAccess[86]" [style=invis]
"TypeAccess[84]" -> "VarAccess[85]" [style=invis]
"Dot[99]" -> "MethodAccess[102]" [style=invis]
"TypeAccess[100]" -> "VarAccess[101]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
110", fillcolor=1]
Edges [label="Total Nr. Edges
143", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
17", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
17", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}