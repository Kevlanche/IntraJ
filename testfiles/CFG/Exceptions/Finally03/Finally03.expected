NodeName: Entry[13] Successors: [TypeAccess[32]]
NodeName: TypeAccess[32] Successors: [VarAccess[33]]
NodeName: VarAccess[33] Successors: [StringLiteral[37]]
NodeName: StringLiteral[37] Successors: [MethodAccess[34]]
NodeName: MethodAccess[34] Successors: [IntegerLiteral[44]]
NodeName: IntegerLiteral[44] Successors: [UncheckedExceptions[43]]
NodeName: UncheckedExceptions[43] Successors: [TypeAccess[53]]
NodeName: TypeAccess[53] Successors: [VarAccess[54]]
NodeName: VarAccess[54] Successors: [StringLiteral[58]]
NodeName: StringLiteral[58] Successors: [MethodAccess[55]]
NodeName: MethodAccess[55] Successors: [TypeAccess[68]]
NodeName: TypeAccess[68] Successors: [VarAccess[69]]
NodeName: VarAccess[69] Successors: [StringLiteral[73]]
NodeName: StringLiteral[73] Successors: [MethodAccess[70]]
NodeName: MethodAccess[70] Successors: [ReturnStmt[42], Exit[21]]
NodeName: ReturnStmt[42] Successors: [Exit[21]]
NodeName: Exit[21] Successors: []

NodeName: Exit[21] Predecessor: [ReturnStmt[42], MethodAccess[70]]
NodeName: ReturnStmt[42] Predecessor: [MethodAccess[70]]
NodeName: MethodAccess[70] Predecessor: [StringLiteral[73]]
NodeName: StringLiteral[73] Predecessor: [VarAccess[69]]
NodeName: VarAccess[69] Predecessor: [TypeAccess[68]]
NodeName: TypeAccess[68] Predecessor: [MethodAccess[55]]
NodeName: MethodAccess[55] Predecessor: [StringLiteral[58]]
NodeName: StringLiteral[58] Predecessor: [VarAccess[54]]
NodeName: VarAccess[54] Predecessor: [TypeAccess[53]]
NodeName: TypeAccess[53] Predecessor: [UncheckedExceptions[43]]
NodeName: UncheckedExceptions[43] Predecessor: [IntegerLiteral[44]]
NodeName: IntegerLiteral[44] Predecessor: [MethodAccess[34]]
NodeName: MethodAccess[34] Predecessor: [StringLiteral[37]]
NodeName: StringLiteral[37] Predecessor: [VarAccess[33]]
NodeName: VarAccess[33] Predecessor: [TypeAccess[32]]
NodeName: TypeAccess[32] Predecessor: [Entry[13]]
NodeName: Entry[13] Predecessor: []
